### DISCUSSION TITLE
Ktls Node

### DISCUSSION THREAD

**Discussion by grijjly** [openssl/openssl#22250] [Q&A]
i tried build node with ./configure --openssl-conf-name=enable-ktls (node v 20.0.8) and installed it //no error
modprobe tls done
i set secureOptions:984 which has 8 984&8=8 //also tried only 8 which is neccesarry ktls flag
i forced cipher to sha128gcm256 and validated also tried with default ciphers
but node does not use ktls 

sudo nano /proc/net/tls_stat: 
TlsCurrTxSw                             0
TlsCurrRxSw                             0
TlsCurrTxDevice                         0
TlsCurrRxDevice                         0
TlsTxSw                                 0
TlsRxSw                                 0

kernel version : 6.2.0-1012-aws 
i can make it work with nginx but on nodejs it does not work, what should i do ?

**Comment by mattcaswell** [comment]
This seems like a question for node (initially at least).
